styleguide/source/assets/scss/03-organisms/_topic-promotion.scss

```diff
@@ -0,0 +1,42 @@
+.ama__topic_promotion {
```

Comment:
Seems like multi-word block level elements should use single hyphen instead of single underline to denote the name (and match the filename). Shouldn't this be `ama__topic-promotion`?

Reply:
Flagging this comment to make the change on the PR for that file (these were included here by accident) - https://github.com/AmericanMedicalAssociation/ama-d8/pull/217